### PR TITLE
Do not use cupy-cuda v8, as it contains breaking changes

### LINKED ISSUE #5193: spacy[cuda100] installs cupy-cuda100 8.0.0b1, which has a breaking change
## How to reproduce the behaviour

```bash
$ pip install spacy[cuda100]==2.2.3
...
$ pip freeze | grep -E '(spacy|cupy)'
cupy-cuda100==8.0.0b1
spacy==2.2.3
```

See the release notes of `cupy-cuda100==8.0.0b1` here: https://github.com/cupy/cupy/releases/tag/v8.0.0b1

> Changes without compatibility
> - Removed cupy.scatter_add, which had been deprecated since CuPy v4. Use cupyx.scatter_add instead.

#### Runtime Error

This results in the following runtime error in my application, but I'm sure would be similarly picked up by your unit tests:

```log
  File "nn_parser.pyx", line 462, in spacy.syntax.nn_parser.Parser.update
  File "_parser_model.pyx", line 246, in spacy.syntax._parser_model.ParserModel.begin_update.finish_parser_update
  File "_parser_model.pyx", line 389, in spacy.syntax._parser_model.ParserStepModel.make_updates
  File "ops.pyx", line 1087, in thinc.neural.ops.CupyOps.scatter_add
AttributeError: module 'cupy' has no attribute 'scatter_add'
```

## Your Environment

* Operating System: Ubuntu 18.04
* Python Version Used: 3.6.9
* spaCy Version Used: 2.2.3
* Environment Information: CUDA version 10.0.103

### PR DESCRIPTION
**Pull request by tommilligan**
## Description

Do not use cupy-cuda v8, as it contains breaking changes. See #5193 for a full description.

### Types of change

Bug fix in the install process.

## Checklist

- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed. _(not done, as no code changes)_
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

Closes #5193

### CODE REVIEW

**Comment by ines** [comment]
@svlandeg I guess spaCy and Thinc should match, though? Not sure what takes precedence if we do `spacy[cuda90]` and spaCy and Thinc define different version ranges here.

Ultimately, it should probably all refer to Thinc and doing `spacy[cuda90]` should just pull in `thinc[cuda90]`. But not sure if that works or if it has unintended side-effects?

**Comment by svlandeg** [comment]
> 
> 
> @svlandeg I guess spaCy and Thinc should match, though? 

I'm not suggesting to limit the range in Thinc, I'm suggesting / hoping that the fix in https://github.com/explosion/thinc/pull/330 means we won't have to restrict the range at all. Sorry, should have been more clear!

Also, that only works if there are no other breaking changes from `cupy-cuda90` 8.0.0 onwards, ofcourse.

**Comment by tommilligan** [comment]
Agree that it doesn't look like v8 will have other breaking changes in it. Maybe it would still be good to add `<9.0.0` though, so this isn't repeated next time?

**Comment by svlandeg** [comment]
Sure, we could do that

**Comment by tommilligan** [comment]
Updated to restrict to `<9.0.0`. Please merge or close as applicable :slightly_smiling_face:

File: .github/contributors/tommilligan.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+7.  Please place an “x” on one of the applicable statement below. Please do NOT
+    mark both statements:
+
+        * [ ] I am signing on behalf of myself as an individual and no other person
+        or entity, including my employer, has or will have rights with respect to my
+        contributions.
+
+        * [ ] I am signing on behalf of my employer or a legal entity and I have the
+        actual authority to contractually bind that entity.
```
Comment (svlandeg) [inline .github/contributors/tommilligan.md]: Could you fill in either of these two options by making it `[x]`?

Comment (tommilligan) [reply .github/contributors/tommilligan.md]: Updated

**Comment by svlandeg** [comment]
(FYI - I don't think it's useful to force push your changes to "hide" the old commits. If anything, that makes the discussion on the PR & the reviews kind of difficult to follow ;-))

**Pull request merged**